apps/www/src/lib/registry/new-york/example/cards/report-issue.svelte

```diff
@@ -73,8 +73,8 @@
 			<div class="grid gap-2">
 				<Label for="security-level">Security Level</Label>
 				<Select.Root selected={securityLevels[1]}>
-					<Select.Trigger id="security-level">
-						<Select.Value placeholder="Select level" />
+					<Select.Trigger>
+						<Select.Value placeholder="Select level" id="area" />
```

Comment:
```suggestion
						<Select.Value placeholder="Select level" />
```

The `value` shouldn't get an id.

Reply:
I agree, but not adding an id yields a browser issue (for without id)